docs/sources/flow/reference/components/otelcol.exporter.loadbalancing.md

```diff
@@ -0,0 +1,256 @@
+---
+title: otelcol.exporter.loadbalancing
+labels:
+  stage: beta
```

Comment:
I labeled this as "beta" because the Otel Collector [also](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/loadbalancingexporter) labeled it as beta